packages/builder/src/pages/builder/portal/apps/onboarding/_components/NamePanel.svelte

```diff
@@ -20,6 +21,10 @@
     if (url.length < 1) {
       return "URL must be provided"
     }
+
+    if (!APP_URL_REGEX.test(url)) {
+      return "Invalid URL"
```

Comment:
Might be better to say "No spaces are allowed" or something, since the only thing this regex tests for is whitespace and it would be nice to inform the user why the URL is invalid to help them fix it.

Reply:
To be honest I think we should use a regex that allows `[0-9a-zA-Z-_]` and nothing else... Paths (`/`) aren't actually allowed in app URLs either - they would probably break it.

Reply:
Done, could you give the most recent commit a look when you get a chance?